sormas-app/app/src/androidTest/java/de/symeda/sormas/app/CaseBackendTest.java

```diff
@@ -272,9 +274,14 @@ public void shouldUpdateCaseAndAssociatedEntitiesOnMove() throws DaoException {
 		Task doneTask = TestEntityCreator.createCaseTask(caze, TaskStatus.DONE, user);
 
 		Case existingCase = caseDao.queryUuidWithEmbedded(caze.getUuid());
-		caze.setDistrict(DatabaseHelper.getDistrictDao().queryUuid(TestHelper.SECOND_DISTRICT_UUID));
-		caze.setCommunity(DatabaseHelper.getCommunityDao().queryUuid(TestHelper.SECOND_COMMUNITY_UUID));
+		District secondDistrict = DatabaseHelper.getDistrictDao().queryUuid(TestHelper.SECOND_DISTRICT_UUID);
```

Comment:
I'd suggest to only set the responsible... instead of both.

This means that we also have to fix the TestEntityCreator.createCase() method to assign responsible... instead of the old infrastructure region,district,community fields that are only relevant when the home/place of stay differs from the responsible region/district/community.